docs/core/project-sdk/msbuild-props.md

```diff
@@ -1582,6 +1568,35 @@ When you use the [MSTest project SDK](../testing/unit-testing-mstest-sdk.md), th
 
 For more information, see [Microsoft.Testing.Platform profile](../testing/unit-testing-mstest-sdk.md#microsofttestingplatform-profile).
 
+## VSTest&ndash;related properties
+
+The following MSBuild properties are documented in this section:
+
+- [IsTestProject](#istestproject)
+- [TestingPlatformDotnetTestSupport](#testingplatformdotnettestsupport)
+- [TestingPlatformShowTestsFailure](#testingplatformshowtestsfailure)
+- [UseVSTest](#usevstest)
+- [MSTestAnalysisMode](#mstestanalysismode)
+
+### IsTestProject
+
+The `IsTestProject` property signifies that a project is a test project. When this property is set to `true`, validation to check if the project references a self-contained executable is disabled. That's because test projects have an `OutputType` of `Exe` but usually call APIs in a referenced executable rather than trying to run. In addition, if a project references a project where `IsTestProject` is set to `true`, the test project isn't validated as an executable reference.
+
+This property is mainly needed for the `dotnet test` scenario and has no impact when using *vstest.console.exe*.
+
+> [!NOTE]
+> If your project specifies the [MSTest SDK](../testing/unit-testing-mstest-sdk.md), you don't need to set this property. It's set automatically. Similarly, this property is set automatically for projects that reference the Microsoft.NET.Test.Sdk NuGet package linked to VSTest.
+
+### TestingPlatformDotnetTestSupport
+
+The `TestingPlatformDotnetTestSupport` property lets you specify whether VSTest is used when you use `dotnet test` to run tests. If you set this property to `true`, VSTest is disabled and all `Microsoft.Testing.Platform` tests are run directly.
+
+If you have a solution that contains VSTest test projects as well as MSTest, NUnit, or XUnit projects, you should make one call per mode (that is, `dotnet test` won't run tests from both VSTest and the newer platforms in one call).
```

Comment:
The wording here is very confusing. This is explained in more details in https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-dotnet-test, but I'm not sure how to summarize the article.

Maybe something like:

```
This property enables testing Microsoft.Testing.Platform application when using the VSTest mode of `dotnet test`.

For more information, see [Testing with'dotnet test'](...).
```